### PR TITLE
week1 homework

### PR DESCRIPTION
**Pull request by sima-milli**

### CODE REVIEW

File: Week1/homework/world-queries.txt [hunk 1]
```diff
@@ -37,4 +37,209 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
...
++-----------------+
+| sum(population) |
++-----------------+
+|      3705025700 |
++-----------------+
+1 row in set (0.01 sec)
+
+8.Which languages are spoken in the Region of ‘South America’
+
+SELECT Language, CountryCode FROM CountryLanguage WHERE CountryCode IN (SELECT Code FROM Country WHERE Region = 'South America');
```
Comment (pcelentano) [inline Week1/homework/world-queries.txt]: nice nested select !

File: Week1/homework/world-queries.txt [hunk 2]
```diff
@@ -37,4 +37,209 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
...
+| Goajiro          | VEN         |
+| Spanish          | VEN         |
+| Warrau           | VEN         |
++------------------+-------------+
+42 rows in set (0.00 sec)
+
+9.What are the languages spoken on all cities named ‘Barcelona’ (you may need to join 3 tables =) )
+
+with join :
+ SELECT Language from CountryLanguage cl JOIN Country co ON cl.CountryCode = co.Code
```
Comment (pcelentano) [inline Week1/homework/world-queries.txt]: nice triple join ! but the inner data scientist in me is wondering why we got spanish twice 🤔 , think you can print the city name and country name as well ?

Comment (sima-milli) [reply Week1/homework/world-queries.txt]: Sure I'll try that 👍 , as for the ERD, I'm working on it and will push it today along with the bonus assignment , I just didn't know you will review the homework already 💃

**Comment by unmeshvrije** [comment]
Please re-open the PR after you make changes

**Pull request closed without merging**